tests/cli/test_registry_cli.py

```diff
@@ -67,3 +67,10 @@ def test_ethpm_registry_commands():
     child_four.expect(r"\(active\)")
     child_four.expect("erc1319://0xabc0000000000000000000000000000000000000:3")
     child_four.expect("yours")
+
+    # test registry explore
+    child_five = pexpect.spawn(
+        f"ethpm registry explore erc1319://0x16763EaE3709e47eE6140507Ff84A61c23B0098A:1"
+    )
+    child_five.expect(ENTRY_DESCRIPTION)
+    child_five.expect("\r\n")
```

Comment:
If you like the current formatting I'll add additional tests for an existing registry live on mainnet, hopefully something with fewer entries 😭

Reply:
Actually after some thought that might be a bit flaky in the future if people add packages and the order is not preserved... maybe we should just add a new test case that releases a local package to a fresh registry

Reply:
Looks good to me! That would be ideal - but I think that would require us to have the tester chain up and running? But, we get a free out of jail card here b/c of the way the registry contract is written it returns the package ids in the same order every time - new ones are at the end of the list. So, you should be safe to use one of the mainnet registries  and just test that the first handful of packages are displayed.